### PR TITLE
feat: implement `EthereumFlow`

### PR DESCRIPTION
**Pull request by danielakhterov**
Signed-off-by: Daniel Akhterov <daniel@launchbadge.com>

**Description**:

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [X] Tested (unit, integration, etc.)

### CODE REVIEW

File: src/EthereumFlow.js [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+        await (
+            await new FileAppendTransaction()
+                .setFileId(fileId)
+                .setContents(callData.subarray(4096))
+                .execute(client)
+        ).getReceipt(client);
+    }
+
+    return fileId;
+}
```
Comment (willparks) [inline src/EthereumFlow.js:221]: Following commit f8836227c219a20661a0fdb377426a381ce08321, unit tests weren't passing. Adding
```
CACHE.ethereumTransactionDataLegacyFromBytes = (bytes) => EthereumTransactionDataLegacy.fromBytes(bytes);
CACHE.ethereumTransactionDataEip1559FromBytes = (bytes) => EthereumTransactionDataEip1559.fromBytes(bytes);
```
to the bottom of the file here lets the tests pass again. Not sure if this is the right spot to assign these, but it seemed like the best fit to as far as I could tell.

Comment (danielakhterov) [reply src/EthereumFlow.js:221]: Heh, forgot to assign those at the bottom of their respective files. Will fix.

File: src/EthereumTransactionData.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                throw new Error(
+                    "CACHE.ethereumTransactionDataLegacyFromBytes is not set"
+                );
+            }
+
+            return CACHE.ethereumTransactionDataLegacyFromBytes(bytes);
+        } else {
+            if (CACHE.ethereumTransactionDataEip1559FromBytes == null) {
+                throw new Error(
+                    "CACHE.ethereumTransactionDataLegacyFromBytes is not set"
```
Comment (willparks) [inline src/EthereumTransactionData.js:33]: Didn't catch this before - looks like this error message is for `[...]LegacyFromBytes`, should be for `[...]Eip1559FromBytes`

Comment (danielakhterov) [reply src/EthereumTransactionData.js]: Good catch, fixed.

**Review by willparks — APPROVED** [review]

**Pull request merged**